translator/opencensus/oc_to_internal.go

```diff
@@ -434,8 +429,6 @@ func ocNodeResourceToInternal(ocNode *occommon.Node, ocResource *ocresource.Reso
 	if len(attrs) != 0 {
 		// TODO: Re-evaluate if we want to construct a map first, or we can construct directly
 		// a slice of AttributeKeyValue.
```

Comment:
This TODO is probably no longer relevant.

Reply:
It is still, this is about the current AttributeMap ctor which accepts now a map, and question is if we should accept a slice of AttributesKeyValues.